v2/api-validator/tests/server-tests/bad-request-body.test.ts

```diff
@@ -41,6 +51,11 @@ describe('Test request bodies missing one required property', () => {
           beforeAll(async () => {
             const badBody = _.cloneDeep(goodBody);
             deleteDeepProperty(badBody, propertyPath);
+
+            if (Object.prototype.hasOwnProperty.call(badBody, 'idempotencyKey')) {
+              badBody['idempotencyKey'] = randomUUID();
+            }
```

Comment:
Same as in the previous PR. Unless something prevents it, use this shorter form:
```javascript
if (badBody.hasOwnProperty('idempotencyKey')) { ... }
```

Reply:
My IDE complained about this ESLint check

`ESLint: Do not access Object.prototype method 'hasOwnProperty' from target object.(no-prototype-builtins)`

Shall I ignore it?